.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
   build:
     machine:
-      image: ubuntu-1604:201903-01
```

Comment:
Had to update this to make the `.` in hostname work in docker-compose. This is now in par with the image we use to build ARM images.